contracts/test/FeeContract.t.sol

```diff
@@ -17,160 +17,6 @@ import { FeeContract } from "../src/FeeContract.sol";
 import { DeployFeeContractScript } from "../script/Fee.s.sol";
 
 /// @title FeeContract Test
-contract FeeContractTest is Test {
```

Comment:
I wonder why this was deleted.

Reply:
I thought that it was redundant as there were the same tests in the `FeeContractUpgradabilityTest`

Reply:
Some tests were not redundant. They have been readded in bf55348.

Reply:
thanks @philippecamacho!!